src/components/tracksContainer/TracksContainer.js

```diff
@@ -1,6 +1,8 @@
 import React, { useState, useEffect, useContext } from 'react';
 import { ShowContext } from '../../contexts/ShowContext';
+import { ToastContainer, toast, Zoom, Bounce } from 'react-toastify';
```

Comment:
Looks like this may not be useful for the purposes of our application as we discussed, but thank you so much for leaving the examples in for us to take a look at.